docs/download.md

```diff
@@ -52,28 +52,29 @@ do not have CUDA forward compatibility (for example, GeForce), CUDA 11.5 or late
 need to ensure the minimum driver (450.80.02) and CUDA toolkit are installed on each Spark node.
 
 ### Verify signature
-* Download the [RAPIDS Accelerator for Apache Spark 22.10.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar)
-  and [RAPIDS Accelerator for Apache Spark 22.10.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar.asc)
+* Download the [RAPIDS Accelerator for Apache Spark 22.12.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.12.0/rapids-4-spark_2.12-22.12.0.jar)
+  and [RAPIDS Accelerator for Apache Spark 22.12.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.12.0/rapids-4-spark_2.12-22.12.0.jar.asc)
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
-* Verify the signature: `gpg --verify rapids-4-spark_2.12-22.10.0.jar.asc rapids-4-spark_2.12-22.10.0.jar`
+* Verify the signature: `gpg --verify rapids-4-spark_2.12-22.12.0.jar.asc rapids-4-spark_2.12-22.12.0.jar`
 
 The output if signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
 New functionality and performance improvements for this release include:
-* Dataproc qualification, profiling, bootstrap and diagnostic tool
-* Databricks custom docker container
-* AQE support on Databricks
-* MultiThreaded Shuffle feature
-* Binary write support for parquet
-* Cast binary to string  
-* Hive parquet table write support
+* Zstandard compression for Parquet and ORC
+* Support for Hive text reading
+* Improved performance on `like` operations
+* Tiered projections for more expressions to optimize performance  
+* Support for mapInArrow, instr and array_remove operations
+* z-ordering capability on Databricks Delta Lake
+* Delta Lake batch writes on Databricks Delta Lake
```

Comment:
This implies we have significant support for Delta Lake writes which is not the case (and it's disabled by default).  Will defer to @sameerz how this should be worded or if it even should be worded at all, but I don't think we should state such a general support for Delta Lake write support in this release.

Reply:
We can remove the `* Delta Lake batch writes on Databricks Delta Lake` since it is a limited feature and behind a config flag.